src/redfish_collector/core/rawCollector.py

```
@@ -171,8 +175,33 @@ async def dataCollector(serverAddress,username,password,templateDir,logLevel):
+                            tokenValue = tokenData['X-Auth-Token']
+                            if tokenData['Location'].startswith('https://'):
+                                logoutURL = tokenData['Location']
+                            else:
+                                logoutURL = "https://%s%s" % (serverAddress,tokenData['Location'])
+                            logging.info("[%s] Get Token Value successfully" % serverAddress)
+                            pass
+                        else:
+                            logging.error("[%s] Can't get X-Auth-Token from response headers" % serverAddress)
+                            return
+                    pass
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)